configuration/config2/SchemaItem.cpp

```diff
@@ -479,6 +479,17 @@ std::shared_ptr<const SchemaItem> SchemaItem::getSchemaRoot() const
 }
 
 
+void SchemaItem::getPath(std::string &path) const
+{
+    path = getProperty("name") + path;
+    if (!m_pParent.expired())
+    {
+        path = "/" + path;
```

Comment:
I'd be surprised if this is the only place we use this path delimiter, let's use a predefined path sep

Reply:
The SchemaItem class is the generic schema class representing an item in a tree of schema items. I defined the '/' as the path separator in this context, so nothing else is considered valid. If a predefined delimiter is still desired, let me know.